pages/extensions/amp_dependencies/__init__.py

```diff
@@ -34,17 +34,18 @@ def __repr__(self):
     return '<AmpDependencies>'
 
   def add(self, name, version=None, type='element'):
-    tag, _version = self._dependencies.get(name, (None, version))
+    if version == None:
+        self._pod.logger.warning('Adding an AMP dependency ({}) without a specific version is not recommended.'.format(name))
```

Comment:
I'm not sure I agree. We should ideally always use the latest version. Having to explicitly define a version should be an exception (because it means there are breaking API changes).

Reply:
Oh we could also remove the warning, but this PR doesn't even add it. I was just rearranging code here.